crates/solidity/inputs/language/src/definition.rs

```diff
@@ -0,0 +1,3854 @@
+                                )
+                            ]
+                        ),
+                        Separated(
+                            name = YulArguments,
+                            separated = NonTerminal(YulExpression),
+                            separator = Terminal([Comma]),
+                            allow_empty = true
+                        ),
+                        Separated(
+                            name = YulIdentifierPaths,
+                            separated = NonTerminal(YulIdentifierPath),
+                            separator = Terminal([Comma])
+                        ),
+                        Separated(
+                            name = YulIdentifierPath,
+                            separated = Terminal([YulIdentifier]),
+                            separator = Terminal([Period])
+                        ),
+                        Token(
+                            name = YulIdentifier,
+                            definitions = [TokenDefinition(scanner = Fragment(RawIdentifier))]
+                        ),
+                        Enum(
+                            name = YulLiteral,
+                            variants = [
+                                Variant(
+                                    name = Boolean,
+                                    fields = (literal =
+                                        Required(Terminal([YulTrueKeyword, YulFalseKeyword])))
+                                ),
+                                Variant(
+                                    name = Decimal,
+                                    fields = (literal = Required(Terminal([YulDecimalLiteral])))
+                                ),
+                                Variant(
+                                    name = Hex,
+                                    fields = (literal = Required(Terminal([YulHexLiteral])))
+                                ),
+                                Variant(
+                                    name = HexString,
+                                    fields = (literal = Required(Terminal([HexStringLiteral])))
+                                ),
+                                Variant(
+                                    name = AsciiString,
+                                    fields = (literal = Required(Terminal([AsciiStringLiteral])))
+                                )
+                            ]
+                        ),
+                        Token(
+                            name = YulDecimalLiteral,
+                            definitions = [TokenDefinition(
+                                scanner = TrailingContext(
+                                    scanner = Choice([
+                                        Atom("0"),
+                                        Sequence([
+                                            Range(inclusive_start = '1', exclusive_end = '9'),
```

Comment:
These `exclusive_end`s should all be `inclusive_end`

Reply:
Good catch. Fixed!